src/components/ebay-dialog/component.js

```diff
@@ -67,8 +68,18 @@ module.exports = {
                     this.emit('dialog-show');
                 } else {
                     bodyScroll.restore();
+                    const activeElement = document.activeElement;
                     this.emit('dialog-close');
 
+                    if (
+                        // Skip restoring focus if the focused element was changed via the dialog-close event
+                        activeElement === document.activeElement &&
+                        // Skip restoring focus if the previously focused element was removed from the DOM.
+                        this._prevFocusEl.parentNode
```

Comment:
`isConnected` would be better as if the previously focused element weren't removed directly it would still have a parent.

Problem is browser support.  Edge and IE don't support this property.

Reply:
Good catch. I'll come up with a solution that works in IE and Edge as well.

Reply:
Fixed using `document.documentElement.contains` instead.

Reply:
@mlrawlings is this solution to your satisfaction? :-) If so, we can merge.